username_api.py

```diff
@@ -73,10 +73,14 @@ def check_username(website, username):
 			'possible': possible,
 		}
 
-	if website in ['pinterest', 'gitlab']:
+	if website in ['pinterest', 'gitlab', 'openhub']:
 		res  = r.get(url)
-		code = 200 if bytes(username, encoding='utf-8') in res.content \
-			else 404
+		if website == 'openhub':
```

Comment:
why do you need to do this?  It returns 404

```
wget -S https://www.openhub.net/accounts/jayvdb123
--2017-12-23 13:25:35--  https://www.openhub.net/accounts/jayvdb123
Resolving www.openhub.net (www.openhub.net)... 192.231.134.139
Connecting to www.openhub.net (www.openhub.net)|192.231.134.139|:443... connected.
HTTP request sent, awaiting response... 
  HTTP/1.1 404 Not Found
```

Reply:
what? It responded 200 when I try. ok